examples/react-native/README.md

```diff
@@ -1,6 +1,4 @@
-This examples shows how to use `react-instantsearch` along with `react-native`.
-
-It was created using `create-react-native-app`.
+This examples shows how to use React InstantSearch along with React Native.
```

Comment:
```suggestion
This example shows how to use React InstantSearch along with React Native.
```

probably needs to be updated in the other ones too?

Reply:
Yes, we can review all the examples' READMEs in another pass.